src/EdozoLogo/assets/maps-logo.svg

```diff
@@ -1 +1 @@
287 2.356 5.364 2.356 2.17 0 3.843-.744 5.022-2.263V26h3.999V4.3h-4v8.029Zm-7.378 9.083c-.806-.837-1.21-1.891-1.21-3.162s.404-2.325 1.21-3.131c.837-.837 1.86-1.24 3.1-1.24s2.263.403 3.069 1.24c.806.806 1.209 1.86 1.209 3.131 0 1.271-.403 2.325-1.21 3.162-.805.806-1.828 1.209-3.068 1.209s-2.263-.403-3.1-1.209Zm28.24 2.666c1.58-1.581 2.387-3.534 2.387-5.828 0-2.294-.806-4.216-2.387-5.797-1.581-1.581-3.534-2.387-5.828-2.387-2.294 0-4.247.806-5.828 2.387-1.581 1.581-2.356 3.503-2.356 5.797 0 2.294.775 4.247 2.356 5.828 1.58 1.581 3.534 2.356 5.828 2.356 2.294 0 4.247-.775 5.828-2.356Zm-8.835-2.759c-.775-.806-1.178-1.829-1.178-3.069s.403-2.263 1.178-3.069c.806-.806 1.798-1.209 3.007-1.209s2.2.403 3.007 1.209c.806.806 1.209 1.829 1.209 3.069s-.403 2.263-1.21 3.069c-.805.806-1.797 1.209-3.006 1.209-1.21 0-2.201-.403-3.007-1.209Zm24.505-8.153V10.5H56.377v3.751h6.231l-6.54 9.083V26h12.244v-3.751H61.43l6.572-9.083Zm15.481 10.912c1.581-1.581 2.387-3.534 2.387-5.828 0-2.294-.806-4.216-2.387-5.797-1.58-1.581-3.534-2.387-5.828-2.387-2.294 0-4.247.806-5.828 2.387-1.58 1.581-2.356 3.503-2.356 5.797 0 2.294.775 4.247 2.356 5.828 1.581 1.581 3.534 2.356 5.828 2.356 2.294 0 4.247-.775 5.828-2.356Zm-8.835-2.759c-.775-.806-1.178-1.829-1.178-3.069s.403-2.263 1.178-3.069c.806-.806 1.798-1.209 3.007-1.209 1.21 0 2.201.403 3.007 1.209s1.21 1.829 1.21 3.069-.404 2.263-1.21 3.069c-.806.806-1.798 1.209-3.007 1.209s-2.2-.403-3.007-1.209Z" fill="#060048"/><path d="m106.523 10.6-4.4 7.282-4.422-7.282h-3.124V26h3.036v-9.966l4.334 7.128h.352l4.334-7.15V26h3.014V10.6h-3.124Zm14.222 5.698c-.858-1.078-2.046-1.606-3.586-1.606-1.474 0-2.728.572-3.784 1.694-1.034 1.122-1.562 2.486-1.562 4.114 0 1.628.528 3.014 1.562 4.136 1.056 1.122 2.31 1.672 3.784 1.672 1.54 0 2.728-.528 3.586-1.606V26h2.838V15h-2.838v1.298Zm-5.236 6.446c-.572-.594-.858-1.342-.858-2.244 0-.902.286-1.65.858-2.222.572-.594 1.298-.88 2.178-.88.88 0 1.606.286 2.178.88.594.572.88 1.32.88 2.222 0 .902-.286 1.65-.88 2.244-.572.572-1.298.858-2.178.858-.88 0-1.606-.286-2.178-.858Zm17.049-8.052c-1.54 0-2.728.528-3.564 1.606V15h-2.838v15.4h2.838v-5.698c.836 1.078 2.024 1.606 3.564 1.606 1.474 0 2.75-.55 3.784-1.672 1.056-1.122 1.584-2.508 1.584-4.136 0-1.628-.528-2.992-1.584-4.114-1.034-1.122-2.31-1.694-3.784-1.694Zm-2.706 8.052c-.572-.594-.858-1.342-.858-2.244 0-.902.286-1.65.858-2.222.572-.594 1.298-.88 2.178-.88.88 0 1.606.286 2.178.88.594.572.88 1.32.88 2.222 0 .902-.286 1.65-.88 2.244-.572.572-1.298.858-2.178.858-.88 0-1.606-.286-2.178-.858Zm12.457-4.62c0-.55.55-.902 1.276-.902.814 0 1.408.374 1.76 1.122l2.42-1.32c-.858-1.496-2.398-2.332-4.18-2.332-1.188 0-2.178.308-2.992.946-.792.616-1.188 1.474-1.188 2.552 0 2.288 2.046 2.97 3.674 3.388 1.122.286 2.002.616 2.002 1.21 0 .638-.506.946-1.518.946-1.122 0-1.826-.484-2.156-1.43l-2.464 1.386c.814 1.738 2.354 2.618 4.62 2.618 1.254 0 2.31-.308 3.146-.902.858-.616 1.276-1.496 1.276-2.618.022-2.354-2.068-3.124-3.674-3.454-1.122-.286-2.002-.616-2.002-1.21Z" fill="#00D4FF"/></svg>
```

Comment:
We are changing width only for maps and insight logo here, but not occupiers, which makes it not correctly aligned in the product switch.